2022-02-04-qubes-4-1-0.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: "Qubes OS 4.1.0 has been released!"
+categories: releases
+download_url: /downloads/
+author: Andrew David Wong
+---
+
+At long last, the Qubes 4.1.0 stable release has arrived!
+
+
+## What's new in Qubes 4.1.0?
+
+In case you still haven't heard, Qubes 4.1.0 includes several major new
+features, each of which is explained in depth in its own article.
+
+
+### [Qubes Architecture Next Steps: The GUI Domain]
```

Comment:
it should be mentioned that it is still experimental and not enabled by default

Reply:
It's stated in the article that this is off by default in 4.1, but can also mention it here.